python/shark_turbine/dynamo/tensor.py

```diff
@@ -40,6 +40,23 @@
 # Factories and device enablement
 ###############################################################################
 
+def get_itemsize(dtype: torch.dtype):
```

Comment:
Fine to do in a followup: There are some dtype tables at the bottom. Can you move this down there and make it into a dict lookup?

Reply:
Ah I missed that detail - I'll move it over.